src/server/game/Entities/GameObject/GameObject.cpp

```diff
@@ -2235,7 +2235,7 @@ void GameObject::BuildValuesUpdate(uint8 updateType, ByteBuffer* data, Player* t
 
             if (index == GAMEOBJECT_DYNAMIC)
             {
-                uint16 dynFlags = 0;
+                uint16 dynFlags = m_uint32Values[GAMEOBJECT_DYNAMIC];
```

Comment:
For some reason, the core completely ignores any `GAMEOBJECT_DYNAMIC` flags set on GOs and _only_ ever sends the ones specifically ORed in inside that method - this is because the dynflags to be sent value is zero initialized instead of using the flag value.

I can move it to a separate PR if you think it necessary.